app/assets/javascripts/components/feedback/feedback_page.es6.jsx

```diff
@@ -0,0 +1,78 @@
+class FeedbackPage extends Component {
+
+  // --------------------------------------------------
+  // Setup
+  // --------------------------------------------------
+  constructor(props) {
+    super(props);
+    this._listener = (state) => this.setState(state);
+  }
+
+  // --------------------------------------------------
+  // Props
+  // --------------------------------------------------
+
+  static get PropTypes() {
+    return {
+      profile: React.PropTypes.object.isRequired,
+    }
+  }
+
+
+  // --------------------------------------------------
+  // Styles
+  // --------------------------------------------------
+  get styles() {
+    return {
+      container: {
+        display: 'flex',
+        flexFlow: 'column',
+        alignItems: 'center',
+        paddingTop: StyleConstants.heights.header,
+      },
+      footer: {
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'flex-end',
+        alignSelf: 'stretch',
+        margin: '12px 0px',
+      },
+      label: {
+        paddingRight: '4px',
+      },
+    };
+  }
+
+  // --------------------------------------------------
+  // Lifecycle
+  // --------------------------------------------------
+  componentWillMount() {
+    this.setState(FeedbackStore.getState());
```

Comment:
If all these lifecycle's made sense, you're well on your way to understanding flux architecture already :+1: